docs/quick-start-guide.md

```diff
@@ -99,6 +99,23 @@ At the moment we have:
 
 Please contribute more! We'd especially like to see a Debian package!
 
+### Testing
```

Comment:
I don't think this should be part of the quick-start guide. It's already too big and is meant to be just the basics for new users to do "hello world".

Reply:
i'll add a developer docs :)